src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4793,6 +4793,26 @@
      * Its children are the name of the member, the assignment literal
      * and the (compile-time constant conditional expression) value.
      *
+     * <p>For example:</p>
+     * <pre>
+     * &#064;MyAnnotation(someField1 = "Hello")
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     *  |--ANNOTATION
```

Comment:
Remove `|--` ,Annotation node should be like root, see examples in other nodes

Reply:
@romani I changed that, please check it now, is it correct?